modules/ReDelay/ReDelay.cpp

```diff
@@ -69,7 +69,7 @@ void handleInput(SokuLib::KeymapManager *base) {
 
 	auto &mgr = SokuLib::getBattleMgr();
 
-	if (base == mgr.leftCharacterManager.keyManager->keymapManager) {
```

Comment:
Does it even work though ?
You are checking for mgr.**right**CharacterManager.keyManager but in the condition it dereferences mgr.**left**CharacterManager.keyManager.